src/libraries/System.Net.Quic/src/System/Net/Quic/Implementations/MsQuic/MsQuicConnection.cs

```diff
@@ -87,6 +87,11 @@ public MsQuicConnection(IPEndPoint localEndPoint, IPEndPoint remoteEndPoint, Saf
                 _stateHandle.Free();
                 throw;
             }
+
+            if (NetEventSource.Log.IsEnabled())
+            {
+                NetEventSource.Info(_state, $"[Connection#{_state.GetHashCode()}] inbound connection created");
```

Comment:
Ohh, this is a prime example where could actually log the handle to be able to easily map it to the msquic logs, see https://github.com/dotnet/runtime/issues/53224:
> on the .NET side I'm wondering if we should show (or log) the handle so it easier to correlate to the native logs.

But that's absolutely not the goal of this PR and you shouldn't worry about it here, just good to be aware of it.

Reply:
Yes, that's indeed is a good example. But let's update it separately, I'd love this PR to be merged sooner to stop blocking others